integration_test/third_party_apps_test.go

```diff
@@ -148,10 +151,9 @@ func runScriptFromScriptsDir(ctx context.Context, logger *logging.DirectoryLogge
 // Installs the agent according to the instructions in a script
 // stored in the scripts directory.
 func installUsingScript(ctx context.Context, logger *logging.DirectoryLogger, vm *gce.VM) (bool, error) {
-	environmentVariables := make(map[string]string)
-	suffix := os.Getenv("REPO_SUFFIX")
-	if suffix != "" {
-		environmentVariables["REPO_SUFFIX"] = suffix
+	environmentVariables := map[string]string{
+		"REPO_SUFFIX": os.Getenv("REPO_SUFFIX"),
```

Comment:
Careful: this changes the semantics. If REPO_SUFFIX is unset, this will actually set it to an empty string. And, while in Go you can't distinguish between "unset" and "set to empty", bash does make that distinction.